chingu-coders/Voyage2-Turtles-02
# color picker

## Purpose
* To provide user with a pop-up color picker with associated HEX/RGB values

## Approach/Fixes
* Creation credit to Dario Corsi (reference: https://codepen.io/dariocorsi/pen/WwOWPE?editors=0010)
* Uses tinycolor.js (reference: https://github.com/bgrins/TinyColor)

## Testing
* [Check out this pull request](https://help.github.com/articles/checking-out-pull-requests-locally/)
* Click the paintbrush icon in the bottom left-hand corner
* Check to see a color picker pop-up box appears
* Move the eyedrop selector to view different colors and their associated RGB values
* Change any of the RGB values directly in their associated inputs (eyedrop selector should move afterwards)
* Click `mode` to switch from RGB to HEX values
* Check to see that the preview box in the upper right-hand corner shows the same color as the eyedrop selector
* Test the hue selector to the right of the color canvas
* Click the paintbrush icon again to hide the pop-up box
